core/chains/evm/forwarders/forwarder_manager.go

```diff
@@ -86,8 +86,9 @@ func (f *FwdMgr) Start(ctx context.Context) error {
 		}
 		if len(fwdrs) != 0 {
 			f.initForwardersCache(ctx, fwdrs)
-			if err = f.subscribeForwardersLogs(fwdrs); err != nil {
-				return err
+			if f.logpoller == evmlogpoller.LogPollerDisabled {
+				f.logger.Warn("log poller is required for contract forwarder to receive AuthorizedSendersChanged logs")
```

Comment:
I'm just preserving the logic the way it is on develop; I've been trying to get confirmation from forwarder owner that this is intentional but haven't heard back yet.  I think the idea is that responding to sender changed authorization events is considered "optional" in that, if it's not working we should still allow the node to start the transaction manager and send transactions.  In fact, I believe it was never working on develop... just always giving this warning message during startup but not responding to these events... also looking for confirmation that this was unintentional (skipping filter subscription due to log poller never being started until after the forwarder is started)... since I'm fixing it under the assumption it was a bug.